src/typings.d.ts

```diff
@@ -1,2 +1,3 @@
+declare module 'aggregate-error';
```

Comment:
```
"devDependencies": {
  "@types/aggregate-error": "1.0.0"
```

means we don't need it here

Reply:
We do, otherwise there's error regards `import`. The type module isn't written well.

Reply:
So then we don't need `@types/aggregate-error`

Reply:
`This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.`. When you use this flag, other issues pop up. It's better this way

Reply:
`declare module 'xxx'` makes it `any` so TS doesn't even use `@types/xxx` any more.

Reply:
But I don't want it to be `any`, I want it to be `AggregateError`. That's why I use both.

Reply:
If you can fix that properly, by all means please do